cypress/integration/Task.spec.js

```diff
@@ -212,11 +207,19 @@ describe('Task component', () => {
     });
     cy.wait(2000);
     cy.get('.form-group').find('button').click();
-    cy.route('PUT', 'http://localhost:8080/api/1.0/tasks/1').then(function() {
+    /* TODO This call is failing with a 404 . Debug on the why. I could trace that the line 221, user_id was being sent
```

Comment:
This is failing because the following Route is not being intercepted, also note that the URL is malformed (`user_id=null`, `process_request_id=null`):
```
http://localhost:8080/api/1.0/tasks?user_id=null&status=ACTIVE&process_request_id=null&include_sub_tasks=1
```
Prior to this PR, it also returned 404, but Cypress did not end the test in a failed state.
![image](https://user-images.githubusercontent.com/8028650/218753628-78de904c-7719-4556-90fe-d8f1853d430e.png)

The following ticket was created to fix this issue: https://processmaker.atlassian.net/browse/FOUR-7412